openjdk.test.jck/src/native/makefile

```diff
@@ -140,7 +140,7 @@ SRC_PATH=$(SRCDIR)/src
 # From jck10 the unix include files are in src/share/lib/jni/include/linux
 
 CC=gcc
-CFLAGS=-fPIC -I$(SOLARIS_PATH) -I$(LINUX_PATH) -I$(SRC_PATH)
+CFLAGS=-fPIC -I$(LINUX_PATH) -I$(SRC_PATH)
```

Comment:
Why Linux path and not solaris path?

Reply:
This is the default CFLAGS for Linux (I moved the Solaris block to it's own IF statement)